### ISSUE TITLE
4.x regression: exports do not immediately update

### ISSUE THREAD

**Issue by nathanfranke** [godotengine/godot#62950]
### Godot version

v4.0.alpha.custom_build [d26442e70]. Not reproducible in 3.4.4

### System information

Arch on 5.18.10-arch1-1 

### Issue description

title

### Steps to reproduce

1) Write the script:

    ```gdscript
    extends Node
    ```

2) (Re)select the node to open it in inspector

    ![image](https://user-images.githubusercontent.com/14253836/178503396-c6055e6a-64b7-41bc-a1a4-898914a85142.png)

3) Add `@export var test: int`. It will not immediately be added to the inspector, even if the script is saved.

    ![image](https://user-images.githubusercontent.com/14253836/178503907-de9fa5f9-1f46-4f4d-91a4-7d4dd9ec98de.png)

4) Remove the line you just added and save. It will be updated with the wrong state:

    ![image](https://user-images.githubusercontent.com/14253836/178504112-da30d533-a1d5-4946-a226-250892bffcbc.png)

### Minimal reproduction project

basically an empty project
[test9.zip](https://github.com/godotengine/godot/files/9093514/test9.zip)

**Comment by kinami-imai** [comment]
This is no longer reproducible in v4.0.3 or the latest master build. In fact, Godot is now overeager when updating the exports: https://github.com/godotengine/godot/issues/78126

I recommend closing the issue, as it is fixed.

**Comment by akien-mga** [comment]
Thanks, closing.
